### ISSUE TITLE
The result issue

### ISSUE THREAD

**Issue by hopkin-lai** [minqukanq/hierarchical-multi-label-text-classification#2]
Now the code works fine, there is no error any more. But the results of the experiment looks terrible to me. Is it a common issue (algorithm itself) or  something wrong with my testing? ( I am also using the original data set in this repository)
```
root - INFO - All Validation set: Loss 1.44963 | AUC 0.354399 | AUPRC 0.000575171
root - INFO - Predict by threshold: Precision 0.000449691, Recall 0.242424, F 0.000897716
root - INFO - Predict by topK:
root - INFO - Top1: Precision 0, Recall 0, F 0
root - INFO - Top2: Precision 0, Recall 0, F 0
root - INFO - Top3: Precision 0, Recall 0, F 0
root - INFO - Top4: Precision 0, Recall 0, F 0
root - INFO - Top5: Precision 0, Recall 0, F 0
root - INFO - Finished Training.
```

**Comment by minqukanq** [comment]
Yes, this is normal.
My implementation is using a very low learning rate and warm-up.
As you continue to learn, your performance will improve. :)

**Comment by minqukanq** [comment]
If the performance does not improve even if you continue to learn, please tell me again. :)

**Comment by minqukanq** [comment]
It may be useful to measure the performance of each layer more specifically by using the first_logits and second_logits in the output of the model.

**Comment by hopkin-lai** [comment]
> It may be useful to measure the performance of each layer more specifically by using the first_logits and second_logits in the output of the model.

**Comment by hopkin-lai** [comment]
Thank u so much !!!  It helped me a lot, I will continue to learn.

**Comment by gumanchang** [comment]
So, epoch is 20? or 50? or even more?
